README.md

```diff
@@ -1 +1,24 @@
-# officer-delta-processor
\ No newline at end of file
+# officer-delta-processor
+
+The Officer Delta Processor is responsible for reading from the officer-delta kafka topic.
+
+Requirements
+------------
+
+To build officer-delta-processor, you will need:
+* [Git](https://git-scm.com/downloads)
+* [Java 11](https://www.oracle.com/uk/java/technologies/javase-jdk11-downloads.html)
+* [Maven](https://maven.apache.org/download.cgi)
+* [Apache Kafka](https://kafka.apache.org/)
+* Internal Companies House core services
+
+You will also need a REST client (e.g. Postman or cURL) if you want to interact with any officer-delta-processor service endpoints.
+
+## Building and Running Locally
+
+1. From the command line, in the same folder as the Makefile run `make clean build`
+1. Configure project environment variables where necessary (see below).
+1. Ensure dependent Companies House services are running within the Companies House developer environment
+1. Start the service in the CHS developer environment
+1. Send a GET request using your REST client to /officer-delta-processor/healthCheck. The response should be 200 OK with status=UP.
```

Comment:
This comment could be made in several places, but I'll keep it here.
Is it usual CH behaviour to include any payload in a healthcheck response? For example, is it possible to have a healthcheck give a 200 with "DOWN" in it?

Reply:
We confirm the service is responding by doing a health check.

Reply:
... but does the healthcheck require a payload of any sort? Or just a 200ok response?
I ask because we (my team) are looking into standardising healthcheck behaviour at the moment & I'm interested in how best to go about this, so this is not criticism but curiosity about current/desired practice.

Reply:
Speaking to Simon Boyd, our health check is a get request returning just the 200 response.
Simon is interested in having a short meeting in near future with you to discuss standardisation.

Reply:
Very happy to have a quick meeting with Simon. If it is only returning the 200, you will want to update your api spec to not mention the ServiceStatus object: https://github.com/companieshouse/officer-delta-processor/pull/2/files#diff-e86fbd38497786da8d475549c2d7c81274c747efed425ef817e8df8aaa0a5004

... but I am happy to leave it if you have a Jira to track this.